docs/resources/aws_connection.md

```diff
@@ -0,0 +1,50 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "clumio_aws_connection Resource - terraform-provider-clumio"
+subcategory: ""
+description: |-
+  Clumio AWS Connection Resource used to connect AWS accounts to Clumio.
+---
+
+# clumio_aws_connection (Resource)
+
+Clumio AWS Connection Resource used to connect AWS accounts to Clumio.
+
+## Example Usage
+
+```terraform
+resource "clumio_aws_connection" "example" {
+  account_native_id           = "aws_account_id"
+  aws_region                  = "aws_region"
+  description                 = "description"
+  protect_asset_types_enabled = ["EBS", "RDS", "DynamoDB", "EC2MSSQL", "S3"]
+  services_enabled            = ["discover", "protect"]
+  organizational_unit_id      = "organizational_unit_id"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **account_native_id** (String) AWS Account Id to connect to Clumio.
+- **aws_region** (String) AWS Region of account.
+
+### Optional
+
+- **description** (String) Clumio AWS Connection Description.
+- **organizational_unit_id** (String) Clumio Organizational Unit Id.
+- **protect_asset_types_enabled** (List of String) The asset types enabled for protect. This is only populated if protect is enabled. Valid values are any of [EBS, RDS, DynamoDB, EC2MSSQL, S3].
+- **services_enabled** (List of String) The services to be enabled for this configuration. Valid values are [discover], [discover, protect]. This is only set when the registration is created, the enabled services are obtained directly from the installed template after that.
+
+### Read-Only
+
+- **clumio_aws_account_id** (String) Clumio AWS AccountId.
+- **clumio_aws_region** (String) Clumio AWS Region.
+- **connection_status** (String) The status of the connection. Possible values include connecting, connected and unlinked.
+- **id** (String) Clumio AWS Connection Id.
+- **namespace** (String) K8S Namespace.
+- **token** (String) The 36-character Clumio AWS integration ID token.
+
+
```

Comment:
Is there supposed to be a section about Importing like this example I found on the Snowflake TF Provider?

https://registry.terraform.io/providers/chanzuckerberg/snowflake/latest/docs/resources/account_grant
<img width="732" alt="Screen Shot 2021-12-29 at 10 53 18 PM" src="https://user-images.githubusercontent.com/50971376/147729650-4b76bd8b-d07f-4cae-a6d6-14fa4be908ec.png">

Reply:
Need to see how to generate this automatically. If not we'll have to add it manually.

Reply:
Added import documentation.